Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -57,6 +57,7 @@ export class azureclitask {
                 tool = tl.tool(tl.which(scriptPath, true));
             }
             this.throwIfError(tl.execSync("az", "--version"));
+            this.useGlobalConfig = tl.getBoolInput("useGlobalConfig");
```

Comment:
tl.getBoolInput("useGlobalConfig") || true

Reply:
We want it to be true for existing RDs

Reply:
done.

Reply:
@bishal-pdMSFT  the syntax you have will always return true. I think this probaby takes a default value

Reply:
yes, you right. the expression is wrong. But we want to be false when it is not set

Reply:
tl.getBoolInput() will return whatever default value present in task.json.